### PR TITLE
Table component issue #80

### PR DESCRIPTION
**Pull request by AdhamAH**
I had some trouble with the old branch so I made a new on

@SomeMoosery I hope I did it right this time

### CODE REVIEW

**Comment by AdhamAH** [comment]
Actually, in this one, I forget to make the changes in the gif! I am sick and sleepy, sorry! that's why it passed I guess

**Comment by tesla809** [comment]
You are doing great @AdhamAH ! Amazing stuff! Feel better and get some rest. We can look over this next time. Brillant code and great suggestion to use material-ui!

**Comment by AdhamAH** [comment]
I could only sleep for 3.5 hours  😢 so I worked on this a little

I am having problems dealing with git and how to pull and push so I hope I did it right.

First I changed the style in DiagnosticContainer and also changed US-EN zone to be undefined, this will take the local system format and apply it I think, please can try and let me know
style

second I removed the date and tabs from table.js and will try to make a toggle component soon 

table

**Comment by AdhamAH** [comment]
Screenshot 2020-03-21 at 08 50 33

File: client/src/css/Table.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+.hr{width: 100px;
+    border: 1px black solid;
+
+}
+@media only screen and (max-width: 414px){
+    .hr{
+        width: 70px;
+    }
+}
+<<<<<<< HEAD
```
Comment (SomeMoosery) [inline client/src/css/Table.css]: It looks like there's a merge conflict tag in here - we just need to rebase off of master, I can add some instructions in my overall comment on the typical strategy to do that!

Comment (AdhamAH) [reply client/src/css/Table.css]: I did push, not sure if that what you wanted, let me know

File: client/src/components/Table.js [hunk 2]
```diff
@@ -1,12 +1,9 @@
 import React from "react";
```
Comment (AdhamAH) [inline client/src/components/Table.js]: @tesla809 I just remimbered that you asked e to use
`import React, {Component} from 'react`
I can do that but may I ask why? what is the difference?

**Pull request merged**